packages/web-app/src/containers/actionBuilder/fundOpportunity/index.tsx

```diff
@@ -1,30 +1,53 @@
-import React from "react";
+import React, { useEffect } from "react";
 import { Label } from "@aragon/ui-components";
-import { Controller, useFormContext } from "react-hook-form";
+import { Controller, useFormContext, useWatch } from "react-hook-form";
 import { useTranslation } from "react-i18next";
 import styled from "styled-components";
-import { CollateralType, FundingSource } from "pages/fundOpportunity";
+import { ActionIndex, CollateralType, FundingSource } from "utils/types";
 
 const fundingSourceOptions: {
     label: string;
     value: FundingSource;
 }[] = [
-    { label: 'DAO treasury', value: 'DAO' },
-    { label: 'Aave borrow', value: 'AAVE' },
-];
+        { label: 'DAO treasury', value: 'DAO' },
```

Comment:
Better as a type, we will need then use these fields in the action encoding

Reply:
I do not understand. `fundingSourceOptions.value` variable already has a 'type'. What are you referring to?

Reply:
OK, yes my bad, the type is defined in another file